community/modules/compute/schedmd-slurm-gcp-v6-partition/variables.tf

```diff
@@ -199,3 +185,14 @@ variable "nodeset_dyn" {
     error_message = "All dynamic nodesets must have a unique name."
   }
 }
+
+# tflint-ignore: terraform_unused_declarations
+variable "network_storage" {
+  description = "DEPRECATED"
+  type        = any
+  default     = null
+  validation {
+    condition     = var.network_storage == null
+    error_message = "network_storage in partition module is deprecated and should be removed."
```

Comment:
Note for future (not this PR). When we add functionality to nodeset, please update this error to reference nodeset. You can close once you have read, so long as you agree.